packages/ckeditor5-ui/src/componentfactory.js

```diff
@@ -77,20 +77,6 @@ export default class ComponentFactory {
 	 * @param {Function} callback The callback that returns the component.
 	 */
 	add( name, callback ) {
-		if ( this.has( name ) ) {
-			/**
-			 * The item already exists in the component factory.
-			 *
-			 * @error componentfactory-item-exists
-			 * @param {String} name The name of the component.
-			 */
-			throw new CKEditorError(
-				'componentfactory-item-exists: The item already exists in the component factory.',
-				this,
-				{ name }
-			);
-		}
```

Comment:
This is a change that probably would need to be discussed.

Reply:
I've scanned the #7803 but I didn't find why this change.

Reply:
OK so this is mostly to @oleq - I don't get from the issue why. The benefits are OK though.

@pomek - please ignore my comments here ;)